public/lib/helpers/timeline_helper.js

```diff
@@ -68,11 +68,11 @@ define(function (require) {
 
       // in kibi, we have the path property of a field
       if (params.labelFieldSequence && params.labelFieldSequence.length) {
-        field = params.labelFieldSequence;
-        value = kibiUtils.getValuesAtPath(hit._source, field);
+        field = params.labelFieldSequence.join('.'); // labelFieldSequence is an array
+        value = kibiUtils.getValuesAtPath(hit._source, params.labelFieldSequence);
       } else if (params.labelField) {
-        field = params.labelField;
-        value = _.get(hit._source, field);
+        field = params.labelField; // labelField is a plain string
+        value = kibiUtils.getValuesAtPath(hit._source, field.split('.'));
```

Comment:
Can there be a scenario that a field name contains a dot ?
And will this case work correctly ? I mean can there be a document like 
```
_source: {
   "aaa.bbb": {
      "ccc": value
   }
}
```